tests/test_type_aware.py

```diff
@@ -116,6 +116,25 @@ def testClientIDRead(self):
                 # length of data should be greater than source data due to encryption overhead
                 self.assertTrue(len(base.memoryview_to_bytes(row[column])) > len(data[column]))
 
+        # check that Acra will save data in the format it passed
+        data['id'] = base.get_random_id()
```

Comment:
this test case already huge and complicated. can we test it in separate function?